### ISSUE TITLE
Maus: Tilt geht schneller als pan

### ISSUE THREAD

**Issue by xwcg** [Acknex/LotterShock#34]

**Comment by MasterQ32** [comment]
Dachte mir schon, dass sowas noch da ist. Werde morgen danach gucken, dass der ganze Input Framerate-Unabhängig ist

**Comment by firoball** [comment]
Player Pan ist so langsam, dass ich aus Platzmangel die Maus anheben und neu platzieren muss. Bitte wieder die alte Geschwindigkeit setzen, das war gut spielbar.

**Comment by MasterQ32** [comment]
Ja, ich kümmer mich drum. Ich weiß nicht mal, warum sich die Geschwindigkeit überhaupt geändert haben sollte...
Gucke mir den Code nochmal an, weil aufm Laptop mit Nubsie macht das Spiel mit den aktuellen Settings massig Laune, was Kamerageschwindigkeit angeht (aber aufm Desktop mit Maus isses ne Katastrophe)

**Comment by MasterQ32** [comment]
Hab nen Fix in commit 6b15ce2047aaa7b1d0c5868146cc2f93cf25e26c hochgeladen, sollte jetzt sowohl für Controller als auch für Maus frameratenunabhängig sein...

Man kann die Sensitivity jetzt auch über die Settings einstellen (auch wenn noch kein Optionsmenü existiert)

**Comment by xwcg** [comment]
jetzt sind zumindest beide achsen gleich schnell, aber zweites problem: die mausgeschwindigkeit wird nicht berücksichtigt, d.h. die mausbewegung funktioniert eher wie ein tastendruck: 1 oder 0

**Comment by MasterQ32** [comment]
> die mausgeschwindigkeit wird nicht berücksichtigt, d.h. die mausbewegung funktioniert eher wie ein tastendruck: 1 oder 0

Ja, das ist `mouse_force` wie es leibt und lebt... Ich probiers mal mit `mickey` aus, aber das schon arg ruckelig

**Comment by MasterQ32** [comment]
Aktualisiert in commit 60c6e8cbd5fa355b9835f6a34f5508578280d5d2.
Bitte um Prüfung!

**Comment by xwcg** [comment]
funktioniert wohl, sensitivity ist nur unglaublich hoch by default, da muss mindestens nen slider zum einstellen rein ^^

**Comment by MasterQ32** [comment]
Ja, die Maus-Sensitivity ist schon echt ne Bitch ^^
Bei jedem anders, ich fand die vorherige sensitivity mit dem thinkpad zum beispiel sehr angenehm, konnte man super mit spielen

**Comment by MasterQ32** [comment]
Die Empfindlichkeit erschlag ich mit #27, von daher sehe ich das als gelöst an

**Comment by xwcg** [comment]
War mal gefixt, aber seit neustem wieder weird. Beschleunigung gibts zwar, aber die wird ab nem bestimmten wert gecappt. Fühlt sich fast noch schlimmer an als vorher
